src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsPrisonerMergedService.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.personalrelationships.profiledetails
+
+import com.microsoft.applicationinsights.TelemetryClient
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.config.trackEvent
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.data.PrisonerMergeDomainEvent
+
+@Service
+class ContactPersonProfileDetailsPrisonerMergedService(
+  private val dpsApiService: ContactPersonProfileDetailsDpsApiService,
+  private val telemetryClient: TelemetryClient,
+) {
+
+  suspend fun prisonerMerged(event: PrisonerMergeDomainEvent) {
+    val bookingId = event.additionalInformation.bookingId
+    val keepingPrisonerNumber = event.additionalInformation.nomsNumber
+    val removedPrisonerNumber = event.additionalInformation.removedNomsNumber
+    val telemetry = mutableMapOf(
+      "bookingId" to bookingId.toString(),
+      "offenderNo" to keepingPrisonerNumber,
+      "removedOffenderNo" to removedPrisonerNumber,
+    )
+
+    runCatching {
+      dpsApiService.mergeProfileDetails(keepingPrisonerNumber, removedPrisonerNumber)
+      telemetryClient.trackEvent("contact-person-profile-details-prisoner-merged", telemetry, null)
```

Comment:
Very simple for us - just call the DPS endpoint so they can do the merge on their side, and they'll raise domain events if the remaining prisoner is changed.